ctest-test/src/t1.rs

```diff
@@ -87,6 +87,7 @@ extern "C" {
     pub static T1static: c_uint;
 }
 
+#[expect(clippy::eq_op)]
```

Comment:
Can we just ignore clippy in this crate's src/lib.rs? `tests/` and `build.rs` have real source but the "source" is mostly dummy files.

Reply:
ok, disabled